src/cmd/api/controllers/video_upload_handler.go

```diff
@@ -18,7 +20,22 @@ type VideoUploadHandler struct {
 	AmqpClient clients.IAmqpClient
 }
 
+var (
```

Comment:
an we regroup them in on var definition and have more accurate names ?

I'm a bit puzzled about those vars, shouldn't we declare them in a separate file ?